examples/mqtt-all.py

```diff
@@ -178,13 +191,16 @@ def main():
     client_id: {device_id}
     port: {args.port}
     topic: {args.topic}
+    username: {args.username}
+    password: {args.password}
 
     Press Ctrl+C to exit!
 
     """
     )
 
     mqtt_client = mqtt.Client(client_id=device_id)
+    mqtt_client.username_pw_set(args.username, args.password)
```

Comment:
I assume `mqtt_client.username_pw_set(None, None)` does not raise an error, otherwise you could include an `if username and password` check

Reply:
yes, that's neater - should do that even if it doesn't raise an error, thanks.